lib/charms/mongodb/v0/mongodb_backups.py

```diff
@@ -157,7 +157,7 @@ def _on_create_backup_action(self, event) -> None:
 
         # cannot create backup if pbm is not ready. This could be due to: resyncing, incompatible,
         # options, incorrect credentials, or already creating a backup
-        pbm_status = self._get_pbm_status()
+        pbm_status = self.get_pbm_status()
```

Comment:
hmmm, how did you port these changes over? I would expect `charmcraft fetch-lib` to also bump the libpatch.

Reply:
In the separate PR I will do that, for the sake of building k8s charm with MongoDB 6 it is not required

Reply:
Why not in this PR?